server/src/org/apache/cloudstack/network/lb/CertServiceImpl.java

```diff
@@ -339,190 +348,182 @@ public SslCertResponse createCertResponse(SslCertVO cert, List<LoadBalancerCertM
-            SecureRandom random = new SecureRandom();
-            Cipher cipher = Cipher.getInstance(pubKey.getAlgorithm());
+            final String data = "ENCRYPT_DATA";
+            final SecureRandom random = new SecureRandom();
+            final Cipher cipher = Cipher.getInstance(pubKey.getAlgorithm());
             cipher.init(Cipher.ENCRYPT_MODE, privKey, random);
-            byte[] encryptedData = cipher.doFinal(data.getBytes());
+            final byte[] encryptedData = cipher.doFinal(data.getBytes());
 
             cipher.init(Cipher.DECRYPT_MODE, pubKey, random);
-            String decreptedData = new String(cipher.doFinal(encryptedData));
-            if (!decreptedData.equals(data))
+            final String decreptedData = new String(cipher.doFinal(encryptedData));
+            if (!decreptedData.equals(data)) {
                 throw new IllegalArgumentException("Bad public-private key");
+            }
 
-        } catch (BadPaddingException e) {
+        } catch (final BadPaddingException e) {
             throw new IllegalArgumentException("Bad public-private key", e);
-        } catch (IllegalBlockSizeException e) {
+        } catch (final IllegalBlockSizeException e) {
             throw new IllegalArgumentException("Bad public-private key", e);
-        } catch (NoSuchPaddingException e) {
+        } catch (final NoSuchPaddingException e) {
             throw new IllegalArgumentException("Bad public-private key", e);
-        } catch (InvalidKeyException e) {
+        } catch (final InvalidKeyException e) {
             throw new IllegalArgumentException("Invalid public-private key", e);
-        } catch (NoSuchAlgorithmException e) {
+        } catch (final NoSuchAlgorithmException e) {
             throw new IllegalArgumentException("Invalid algorithm for public-private key", e);
         }
     }
 
-    private void validateChain(List<Certificate> chain, Certificate cert) {
+    private void validateChain(final List<Certificate> chain, final Certificate cert) {
 
-        List<Certificate> certs = new ArrayList<Certificate>();
-        Set<TrustAnchor> anchors = new HashSet<TrustAnchor>();
+        final List<Certificate> certs = new ArrayList<Certificate>();
+        final Set<TrustAnchor> anchors = new HashSet<TrustAnchor>();
 
         certs.add(cert); // adding for self signed certs
         certs.addAll(chain);
 
-        for (Certificate c : certs) {
-            if (!(c instanceof X509Certificate))
+        for (final Certificate c : certs) {
+            if (!(c instanceof X509Certificate)) {
                 throw new IllegalArgumentException("Invalid chain format. Expected X509 certificate");
+            }
 
-            X509Certificate xCert = (X509Certificate)c;
+            final X509Certificate xCert = (X509Certificate)c;
 
-            Principal subject = xCert.getSubjectDN();
-            Principal issuer = xCert.getIssuerDN();
+            xCert.getSubjectDN();
+            xCert.getIssuerDN();
```

Comment:
What is the purpose of these two lines?   Information is being retrieved, but not used.